bytecomp/symtable.ml

```diff
@@ -220,7 +220,10 @@ let rec transl_const = function
         fields;
       block
   | Const_float_array fields ->
-      Obj.repr(Array.of_list(List.map (fun f -> float_of_string f) fields))
+      let res = Array.Floatarray.create (List.length fields) in
+      List.iteri (fun i f -> Array.Floatarray.set res i (float_of_string f))
+        fields;
+      Obj.repr res
```

Comment:
Two things:
1. Can we change `Array.Floatarray` to `Float.Array`?  (Then we can also add `Float.pi` which will close another GPR too.)
2. This is `Float.Array.of_list`, let's just move the code to the stdlib.

Reply:
Not yet, because there is no `Float` module in the stdlib. But I agree this naming is temporary.

Reply:
Is there a reason we can't introduce a `Float` module?  Otherwise we'll be stuck with `Array.Floatarray` for evermore, no?

Reply:
No, in this PR it contains only the minimum needed to patch the compiler and to experiment with the new type. No reason we can't move or duplicate it to a proper stdlib module when we decide the option should be the standard.

Reply:
OK